### PR TITLE
Update BlackHole connector to transactional API

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by kokosing** [comment]
looks good to me

File: presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnector.java [hunk 1]
```diff
@@ -51,7 +53,19 @@ public BlackHoleConnector(BlackHoleMetadata metadata,
...
-    public ConnectorMetadata getMetadata()
+    public ConnectorTransactionHandle beginTransaction(IsolationLevel isolationLevel, boolean readOnly)
+    {
+        return BlackHoleTransactionHandle.INSTANCE;
+    }
+
+    @Override
+    public boolean isSingleStatementWritesOnly()
+    {
+        return true;
```
Comment (electrum) [inline presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnector.java]: @erichwang should this be false?

Comment (dain) [inline presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnector.java]: Add a todo "implement real transactions"

**Comment by dain** [comment]
looks good

**Pull request closed without merging**